### PR TITLE
Increase visibility of partnerships on homepage

### PR DESCRIPTION
**Pull request by clementbiron**
That changset:
- reverse how it works and instances sections
- use smaller card and 3 columns layout for instances
- fix maintainer image ratio and improve UI

### CODE REVIEW

**Comment by MattiSG** [comment]
A screenshot would help in understanding the expected change :)

**Comment by MattiSG** [comment]
(I edited the name of this PR as it is not a “refactor” in the sense that it would be a different, better implementation of the same functionality: it adds a feature that is pushing for partners 🙂

**Comment by clementbiron** [comment]
image

File: locales/en/instances.json [hunk 1]
```diff
@@ -7,7 +7,8 @@
   "language": "Language",
   "language_other": "Languages",
   "services": "Services",
-  "title": "We create partnerships to cover different industries, languages and jurisdictions",
+  "title": "We establish partnerships to cover different industries, languages and jurisdictions",
+  "subtitle": "We are constantly looking for new partners to track more services. <a href=\"mailto:{{ contactEmail }}?subject=Partnership&body=Dear%20Open%20Terms%20Archive%20team%2C%0A%0AI%20represent%20____%20and%20am%20interested%20in%20tracking%20documents%20from%20services%20that%20operate%20in%3A%0A%0A-%20industry%3A%20_____%0A-%20language%3A%20_____%0A-%20jurisdiction%3A%20_____%0A%0AWe%20could%20allocate%20___%20hours%20per%20week%20to%20target%2C%20maintain%20and%20analyse%20these%20documents.%0A%0AOur%20team%20would%20be%20comprised%20of%20the%20following%20people%20and%20expertise%3A%0A%0A-%20____%0A-%20____%0A%0AThanks%2C\">Contact us</a> to start collaborating!",
```
Comment (MattiSG) [inline locales/en/instances.json:11]:
```suggestion
  "subtitle": "We are constantly looking for new partners to track more services. <br/><a href=\"mailto:{{ contactEmail }}?subject=Partnership&body=Dear%20Open%20Terms%20Archive%20team%2C%0A%0AI%20represent%20____%20and%20am%20interested%20in%20tracking%20documents%20from%20services%20that%20operate%20in%3A%0A%0A-%20industry%3A%20_____%0A-%20language%3A%20_____%0A-%20jurisdiction%3A%20_____%0A%0AWe%20could%20allocate%20___%20hours%20per%20week%20to%20target%2C%20maintain%20and%20analyse%20these%20documents.%0A%0AOur%20team%20would%20be%20comprised%20of%20the%20following%20people%20and%20expertise%3A%0A%0A-%20____%0A-%20____%0A%0AThanks%2C\">Contact us</a> to start collaborating!",
```

File: locales/fr/instances.json [hunk 2]
```diff
@@ -7,7 +7,8 @@
   "language": "Langue",
   "language_other": "Langues",
   "services": "Services",
-  "title": "Nous créons des partenariats pour couvrir différentes industries, langues et juridictions",
+  "title": "Nous établissons des partenariats pour couvrir différentes industries, langues et juridictions",
+  "subtitle": "Nous sommes constamment à la recherche de nouveaux partenaires pour suivre plus de services. <a href=\"mailto:{{ contactEmail }}?subject=Partnership&body=Ch%C3%A8re%20%C3%A9quipe%20de%20d%27Open%20Terms%20Archive%2C%0A%0AJe%20repr%C3%A9sente%20____%20et%20je%20suis%20int%C3%A9ress%C3%A9%20par%20le%20suivi%20de%20documents%20provenant%20de%20services%20qui%20op%C3%A8rent%20dans%20%3A%0A%0A-%20l%27industrie%20%3A%20_____%0A-%20la%20langue%20%3A%20_____%0A-%20la%20juridiction%20%3A%20_____%0A%0ANous%20pourrions%20consacrer%20___%20heures%20par%20semaine%20au%20ciblage%2C%20%C3%A0%20la%20maintenance%20et%20%C3%A0%20l%27analyse%20de%20ces%20documents.%0A%0ANotre%20%C3%A9quipe%20serait%20compos%C3%A9e%20des%20personnes%20et%20des%20comp%C3%A9tences%20suivantes%20%3A%0A%0A-%20____%0A-%20____%0A%0ANous%20vous%20remercions%2C\">Contactez nous</a> pour commencer à collaborer !",
```
Comment (MattiSG) [inline locales/fr/instances.json:11]:
```suggestion
  "subtitle": "Nous cherchons constamment de nouveaux partenaires pour suivre plus de services. <br/><a href=\"mailto:{{ contactEmail }}?subject=Partnership&body=Ch%C3%A8re%20%C3%A9quipe%20de%20d%27Open%20Terms%20Archive%2C%0A%0AJe%20repr%C3%A9sente%20____%20et%20je%20suis%20int%C3%A9ress%C3%A9%20par%20le%20suivi%20de%20documents%20provenant%20de%20services%20qui%20op%C3%A8rent%20dans%20%3A%0A%0A-%20l%27industrie%20%3A%20_____%0A-%20la%20langue%20%3A%20_____%0A-%20la%20juridiction%20%3A%20_____%0A%0ANous%20pourrions%20consacrer%20___%20heures%20par%20semaine%20au%20ciblage%2C%20%C3%A0%20la%20maintenance%20et%20%C3%A0%20l%27analyse%20de%20ces%20documents.%0A%0ANotre%20%C3%A9quipe%20serait%20compos%C3%A9e%20des%20personnes%20et%20des%20comp%C3%A9tences%20suivantes%20%3A%0A%0A-%20____%0A-%20____%0A%0ANous%20vous%20remercions%2C\">Contactez-nous</a> pour collaborer !",
```

Comment (MattiSG) [inline locales/fr/instances.json:11]: Small typo + gender neutrality.
```
Chère équipe Open Terms Archive,

Je représente ____ et aimerais suivre des documents provenant de services qui opèrent dans :

- l'industrie : _____
- la langue : _____
- la juridiction : _____

Nous pourrions consacrer ___ heures par semaine au ciblage, à la maintenance et à l'analyse de ces documents.

Notre équipe serait composée des personnes et des domaines d'expertise suivants :

- ____
- ____

Merci,
```

**Review by MattiSG — APPROVED** [review]
I'd like my suggestions to be taken into account, but approve now as I am unlikely to be able to provide a last review, and the existing changeset is already acceptable 🙂 👍

**Comment by clementbiron** [comment]
Thanks for your last suggestions, I have taken it into account 👍

**Pull request merged**